FirebaseAppCheck/Sources/AppAttestProvider/Storage/FIRAppAttestArtifactStorage.h

```diff
@@ -24,16 +24,20 @@ NS_ASSUME_NONNULL_BEGIN
 /// Check token obtained with App Attest provider.
 @protocol FIRAppAttestArtifactStorageProtocol <NSObject>
 
-/// Set the artifact.
+/// Set the artifact. An artifact previously set for *any* key ID will replace the new one with the
+/// new key ID. The storage always stores as single artifact.
```

Comment:
nit(typo): I think it should be- The storage always stores **a** single artifact.